WORKSPACE

```diff
@@ -291,6 +291,8 @@ setup_legacy_bzlmodrio_ni_cpp_dependencies()
 
 http_archive(
     name = "bzlmodrio-opencv",
+    patch_args = ["-p1"],
+    patches = ["//:opencv.patch"],
```

Comment:
Are we really sticking patch files at the root of the workspace?

Reply:
No, this needs to be upstreamed before submitting, and the patch removed.  We own this repo.  But that takes time to do, so I'm patching to see if I got it all working.

Reply:
Merged upstream now.